spec/functions/since_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+require 'functions/since.rb'
+
+RSpec.describe Since, type: :function do
+  subject { Since }
+
+  it {
+    is_expected.to respond_to(:calculate_units)
+  }
+
+  describe "#calculate_units" do
+    let(:calculate_units) { Since.calculate_units }
+    let(:from) { Time.new(2001,01,01) }
+    let(:to) { Time.new(2021,01,01) }
```

Comment:
it is hard to read tests if dates are so in past. Do not use such a big difference